framework/source/class/qx/ui/form/Form.js

```diff
@@ -291,6 +291,26 @@ qx.Class.define("qx.ui.form.Form",
     },
 
 
+    /**
+     * Return an item by name.
+     *
+     * @param name {string} Item name.
+     * @return {qx.ui.form.IForm|null} The form item or null.
+     */
+    getItem : function(name) {
+      for (var i = 0; i < this.__groups.length; i++) {
+        var group = this.__groups[i];
+        for (var j = 0; j < group.names.length; j++) {
+          if (group.names[j] === name) {
+            return group.items[j];
+          }
+        }
+      }
+
+      return null;
```

Comment:
Calling code should decide what to do when no item is found. IMHO exceptions should only be thrown in exceptional cases. Also, I do not think it is an error to not find an item. If one tries to access a property of null by supplying a wrong name, then the browser will notify you of your mistake. In the meantime generic code can iterate form items without worrying getItem to assume proper outcome:
```
var describeItem = function(item) {...};
var describeItems = function(form, names) {
names.forEach(function(name) {
  var item = form.getItem(name);
  if (null === item) {
    form.warn("Item " + name + " is not found in this form.");
  } else {
    describeItem(item);
  }
});
```